infra/gcp/terraform/modules/oci-proxy/cloud-armor.tf

```diff
@@ -23,9 +23,11 @@ resource "google_compute_security_policy" "cloud-armor" {
 
   # apply rate limits
   rule {
-    action      = "throttle"
-    description = "Default rule, throttle traffic"
```

Comment:
You _must_ have a default rule with this specific priority (The highest value / last applied rule), not doing so is invalid.

We still want throttling to be applied first, however, so garbage requests are factored into throttling.

Reply:
It failed to deploy for this reason:
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-k8s-deploy-registry/1811794309389750272

> googleapi: Error 400: Invalid value for field 'priority': '2147483647'. Cannot remove the default rule., invalid

https://stackoverflow.com/a/60401055

Reply:
https://github.com/kubernetes/k8s.io/pull/6979